src/main/resources/liquibase/scripts/faculty.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset FeralHorse:1
+
+CREATE INDEX faculty_name_color_index ON faculty (name, color)
```

Comment:
```suggestion
CREATE INDEX faculty_name_color_index ON faculties (name, color)
```
не во множественном разве?